torch/export/_trace.py

```diff
@@ -652,9 +652,16 @@ def forward(self, *args, **kwargs):
         ep_non_strict = _export_non_strict(
             f, fake_args, fake_kwargs, fake_params_buffers, transform=_tuplify_outputs
         )
-        range_constraints, equality_constraints = make_constraints(
-            fake_mode, src_equalities, original_signature, ep_non_strict.gm
-        )
+        try:
+            range_constraints = make_constraints(
+                fake_mode,
+                equalities_inputs,
+                original_signature,
+                ep_non_strict.gm,
+            )
+        except (ConstraintViolationError, ValueRangeError) as e:
+            raise UserError(UserErrorType.CONSTRAINT_VIOLATION, str(e))  # noqa: TRY200
```

Comment:
mfff, I'm not a huge fan of suppressing the inner exception here haha, especially when I'm debugging problems I often need the inner backtrace. Consider with_traceback instead of chaining if you think chaining is ugly

Reply:
OK, will TAL.

Reply:
Not planning to do anything here in this PR.